superset/db_engine_specs/trino.py

```diff
@@ -216,6 +216,8 @@ def handle_cursor(cls, cursor: Cursor, query: Query) -> None:
         if tracking_url := cls.get_tracking_url(cursor):
             query.tracking_url = tracking_url
 
+        db.session.commit()
```

Comment:
Maybe move the global override to the line?

```suggestion
        db.session.commit()  # pylint: disable=consider-using-transaction
```